demonstrations/tutorial_vqe_vqd.py

```diff
@@ -0,0 +1,224 @@
+r"""How to implement VQD with PennyLane
+===============================================================
+
+Finding eigenvalues of an operator is a key task in quantum computing. Algorithms like VQE are used to find the smallest
+one, but sometimes we are interested in other eigenvalues. This how-to shows you how to implement  Variational
+Quantum Deflation (VQD) in PennyLane
+and find the first excited state energy of the hydrogen molecule. To benefit the most from this tutorial, we recommend
+a familiarization with the `Variational Quantum Eigensolver (VQE) <https://pennylane.ai/qml/demos/tutorial_vqe/>`_ tutorial.
+
+.. figure:: ../_static/demonstration_assets/vqe_vqd/how_to_vqd_pennylane_opengraph.png
+    :align: center
+    :width: 50%
```

Comment:
```suggestion
    :width: 70%
```
As approved by the Graphic Design Team.